validators/email-question.validator.js

```diff
@@ -0,0 +1,32 @@
+const Joi = require('joi');
+const { emailRegExp } = require('../consts/regexp');
+
+const questionInputValidator = Joi.object({
+  senderName: Joi.string()
+    .min(2)
+    .max(30)
+    .trim()
+    .required(),
+  text: Joi.string()
+    .min(2)
+    .max(700)
+    .trim()
+    .required(),
+  email: Joi.string()
+    .trim()
+    .regex(emailRegExp)
+    .required(),
+  language: Joi.number()
+    .valid(0, 1)
+    .default(0)
```

Comment:
Have you checked if it sets the default value?

Reply:
fixed, delete this line because it doesn't need